test/mini-oidc/go.mod

```diff
@@ -1,13 +1,15 @@
 module github.com/canonical/lxd/test/mini-oidc
 
-go 1.20
+go 1.21
+
+toolchain go1.22.0
```

Comment:
It seems odd that the toolchain version is 1.22 but the go version is 1.21.

Reply:
Since tests have passed with this `toolchain` directive, I think we could even drop the `go get toolchain@none` from the `Makefile`.  The `toolchain` directive was removed in commit 7baaf2d1d72f7b20ad0 because some of our tools didn't like it back then.